docs/plugins/amp/rules-use.md

```
@@ -0,0 +1,362 @@
+````markdown
+{{instructions}}
+Follow these coding standards:
+
+- Use TypeScript for all new code
+- Write comprehensive tests
+- Document all public APIs
+  {{/instructions}}
+
+{{examples}}
+
+```typescript
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)